internal/api/v1/application/importgit.go

```diff
@@ -131,9 +131,9 @@ func (hc Controller) ImportGit(c *gin.Context) apierror.APIErrors {
 		return apierror.InternalError(err, "creating an S3 manager")
 	}
 
-	username := requestctx.User(ctx).Username
+	user := requestctx.User(ctx)
 	blobUID, err := manager.Upload(ctx, tarball, map[string]string{
-		"app": name, "namespace": namespace, "username": username,
+		"app": name, "namespace": namespace, "username": user.Username,
```

Comment:
These changes are no longer needed? (this and anywhere where you switched to pass a User object instead of just the name).

Reply:
I'm not sure, it complicates this PR for no good reason. Maybe leave it out?

Reply:
Done!